index-handlers/src/main/java/no/sikt/nva/nvi/index/SearchNviCandidatesHandler.java

```diff
@@ -171,6 +176,16 @@ private static String extractQueryParamTitle(RequestInfo requestInfo) {
             .getOrDefault(QUERY_PARAM_TITLE, null);
     }
 
+    private static String extractQueryParamContributor(RequestInfo requestInfo) {
```

Comment:
If these parameters don't have a default value, there is no need to use `.getOrDefault()`, you can just use `get()`:
`return requestInfo.getQueryParameters(QUERY_PARAM_CONTRIBUTOR).get()`